Assignment2/belenfh999-fromRDFtoJSON-LD.jsonld

```diff
@@ -1,43 +1,62 @@
 {
-	"@context":{
-		"vocab":"http://temperaturesystems.com/Products/#",
-		"Class01":"vocab:Class01",
-		"Sensor029":"Class01:Sensor029",
-		"Computer101":"Class01:Computer101",
-		"Measurement8401":"Sensor029:Measurement8401",
-		"xsd": "http://www.w3.org/2001/XMLSchema#",
-		"User10A":"Computer101:User10A"
-	},
-	"Class01":{					   
-		"@id":"http://temperaturesystems.com/Products/Class01",
-		"@type":"vocab:Class01",
-      		"Sensor029": {
-			"@id":"http://temperaturesystems.com/Products/Class01/Sensor029",
-      			"@type":"Class01:Sensor029",
-			"Measurement8401":{
-				"@id":"http://temperaturesystems.com/Products/Class01/Sensor029/Measurement8401",
-     				"@type":"Sensor029:Measurement8401",
-          			"temperature":{
-					"@value":"29",
-					"@type":"xsd:integer"
-				},
-              			"atTime":{
-					"@value": "2010-06-12T12:00:12",
-       					"@type": "xsd:dateTime"
-        			}
-			}	
-		},
-		"Computer101":{
-			"@id":"http://temperaturesystems.com/Products/Class01/Computer101",
-      			"@type":"Class01:Computer101",
-      			"User10A":{ 
-				"@id":"http://temperaturesystems.com/Products/Class01/Computer101/User10A",
-      				"@type":"Computer101:User10A",
-          			"name":{
-					"@value":"Pedro",
-					"@type":"xsd:string"
-				}
-        		}
-    		}
-	}
- }
+  "@context": {
+    "vocab": "http://temperaturesystems.com/Products/",
+    "properties": "http://www.temperaturesystems.com/Properties/",
+    "Class01": "vocab:Class01",
+    "Sensor029": "Class01:Sensor029",
+    "Computer101": "Class01:Computer101",
+    "Measurement8401": "Sensor029:Measurement8401",
+    "User10A": "Computer101:User10A",
+    "includes": "properties:includes",
+    "hasMeasurement": "properties:hasMeasurement",
+    "temperature": "properties:temperature",
+    "atTime": "properties:atTime",
+    "hasOwner": "properties:hasOwner",
+    "name": "properties:name"
+  },

```

Comment:
Please, restore the xsd prefix definition.

Reply:
Now it is back, sorry for misunderstanding  your request